src/main/java/io/xpring/xrpl/model/XRPPathElement.java

```diff
@@ -0,0 +1,29 @@
+package io.xpring.xrpl.model;
+
+import org.xrpl.rpc.v1.Payment.PathElement;
+import org.immutables.value.Value;
+
+import java.util.Optional;
+
+/**
+ * A path step in an XRP Ledger Path.
+ * @see "https://xrpl.org/paths.html#path-steps"
+ */
+@Value.Immutable
+public interface XRPPathElement {
+    static ImmutableXRPPathElement.Builder builder() {
+        return ImmutableXRPPathElement.builder();
+    }
+
+    String account();
+    XRPCurrency currency();
+    String issuer();
+
```

Comment:
Can you javadoc this class?

Reply:
documented.  So I wasn't totally clear on what to write for these fields, so I stole from the information here: https://xrpl.org/paths.html#path-steps.  It's a bit verbose, and we also didn't document most of the JS analogues... so we may need to decide on a standard set of documentation and make this consistent throughout the SDKs.  For the rest of these Java PRs, I'll err on the side of verbose and try to document everything, and then can pare down and standardize from there.